Assets/Scripts/SpawnPoint.cs

```diff
@@ -29,6 +36,8 @@ void Update()
     private void SpawnEnemy()
     {
         Vector3 position = transform.position;
-        Instantiate(enemies[_random.Next(enemies.Length - 1)], position, Quaternion.Euler(new Vector3(0f, 0f, 0f)));
+        var map = GameObject.Find("Map");
+        var mapController = (MapGenerator) map.GetComponent(typeof(MapGenerator));
+        mapController.AddEnemy((int)position.x, (int)position.y);
     }
 }
```

Comment:
General question regarding coding guidelines 
Do we do new line at end of file?
@zicons

Reply:
yes, and fixed

Reply:
you forgot new line here